scripts/htmlDependencies.R

```diff
@@ -20,6 +20,7 @@ ignore <- capture.output({
   pak::pkg_install(c(
     "rstudio/bslib@main",
     "rstudio/shiny@main",
+    "rstudio/sass@main",
```

Comment:
I just wanted to make sure we were getting the right fonts since we do some theme re-compilation. I don't think it actually changed any of the files, though so we could take it out. (Or leave it in until the next sass version is released, idk)

Reply:
I'm going to leave this in for now. Installing `sass` takes long enough that I'm sure we won't forget to revisit it 😉